dask/array/core.py

```diff
@@ -1227,6 +1227,33 @@ def from_array(x, chunks, name=None, lock=False):
     return Array(merge({name: x}, dsk), name, chunks, dtype=x.dtype)
 
 
+def from_value(value, shape, dtype=None, name=None):
```

Comment:
I agree that we could probably get rid of the `name` arguments on most dask array functions, but I would actually keep them on the constructors (`from_array` and `from_value`), because they can actually be useful there to clearly identify how the graph was constructed. For example, we use `name` in `from_array` with xray to add file and variable identifiers. Once stuff is already in dask, there's no much point in adding names but it's useful up to that point. Actually, for that matter, maybe we don't need it in `from_value`, given that you can simply edit the name on the input `Value`  objects if you want to make the graph more inspectable.

I think `from_value` is pretty descriptive, supposing you're aware of dask imperative. `from_imperative` and `from_dask_value` both seem quite verbose. But I'm certainly open to other suggestions...